static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -18,6 +18,7 @@
     }
 
     .adverts__row > * + *::before,
+    &.adverts--legacy-single .adverts__row > .advert::after,
```

Comment:
My head is spinning. Can this be moved to the template so you will only add class, something like `adverts__row--legacy` and it will have only after pseudo element in it?